cumulusci/core/source/__init__.py

```diff
@@ -3,7 +3,11 @@
 
 
 class NullSource:
```

Comment:
Can you educate me more on this? The Class name makes it evident however would love to know more for my knowledge, why not handle this in cumulusci defaults? or init_options?

Reply:
Yeah, let' discuss tomorrow!